clusterloader2/pkg/prometheus/manifests/kubemark/kube-apiserver-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
Apiserver itself creates a default k8s service. Can't we use that one?

Reply:
As discussed, this service points to the Kubemark ApiServer, which runs outside of the main cluster.
Added a comment to explain that.